### DISCUSSION TITLE
is there any difference between m3u and xtreme within the inputs section?

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#107] [General]
am new to this project so is going to take me some time to figure how it all works but i wanted to know if there is any difference between m3u and xtreme within the inputs section

**Comment by euzu** [comment]
yes  there is. M3u input is used for m3u playlist urls. Xtream input for xtream api access.
 Both are different apis from the iptv provider.

**Reply by aniel300** [reply]
in terms of sync performance, etc is there a benefit from using one over the other?

**Comment by euzu** [comment]
First at all you need to know which access type your provider gives you.
Xtream Codes ia better than m3u playlist if you have live tv, vod and series. If ypu are only interested in live tv i think m3u is better.

But in the end you define the output format. You can convert them to your desired output format.  But there are some limitations.
If you have xtream input and m3u output you loose series. because this is technically difficult. You can activate fetching series which will result in a provider ban.
ıf you have m3u input and xtream output you will have vod and series as vod.

**Reply by aniel300** [reply]
ohh that is not good. i will xc as input and output then. is that the best combination?

**Comment by euzu** [comment]
yes best is xc in xc out.
